src/main/java/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImpl.java

```diff
@@ -435,6 +436,7 @@ public static class UploadedKeyStoreSource extends KeyStoreSource implements Ser
         @SuppressWarnings("unused") // by stapler
         @Deprecated
         public UploadedKeyStoreSource(String uploadedKeystore) {
+            ensureNotRunningInFIPSMode();
```

Comment:
other plugins are [using this code directly](https://github.com/search?q=org%3Ajenkinsci++%22new+CertificateCredentialsImpl.UploadedKeyStoreSource%22+language%3AJava&type=code), so we can not just not register the extension.
Whilst this will break other plugins, those other plugins are not Compliant and are usign a byte[] that is a PKCS#12 cert - so they are not compliant anyway.

I am not intending to fix /adapt any of these plugins.